docs/source/API/core/view/view_alloc.rst

```diff
@@ -32,14 +32,20 @@ Create View allocation parameter bundle from argument list. Valid argument list
 Description
 -----------
 
-.. cppkokkos:function:: template <class... Args> impl_defined view_alloc(Args const&... args)
+.. cppkokkos:function:: template <class... Args> ALLOC_PROP view_alloc(Args const&... args)
 
    Create View allocation parameter bundle from argument list.
 
    ``args`` : Cannot contain a pointer to memory.
 
-.. cppkokkos:function:: template <class... Args> impl_defined view_wrap(Args const&... args)
+.. cppkokkos:function:: template <class... Args> ALLOC_PROP view_wrap(Args const&... args)
 
    Create View allocation parameter bundle from argument list.
 
    ``args`` : Can only be a pointer to memory.
+
+.. cppkokkos:type:: ALLOC_PROP
+
+   :cppkokkos:type:`ALLOC_PROP` is a special, unspellable implementation-defined type that is returned by :cppkokkos:func:`view_alloc`
```

Comment:
Maybe a bit of explanation around "unspellable" ... ?  Here, the intended meaning is that `ALLOC_PROP` represents an implementation concept, and not an actual feature of the language.  But don't let this comment stop the wheels of progress.
Suggestions to terms in the place of "unspellalbe" -- "implementation defined type" ; "exposition only"; [cpp ref example](https://en.cppreference.com/w/cpp/container/node_handle)